src/db/series.rs

```diff
@@ -74,7 +79,7 @@ pub struct SeriesStatus {
 
 #[derive(Default, Serialize, Deserialize, Clone)]
 pub struct SeasonStatus {
-    pub episodes: BTreeMap<u32, EpisodeStatus>,
+    pub episodes: BTreeMap<String, EpisodeStatus>,
```

Comment:
Because sorting here will no longer be numeric, we might want to do some kind of custom type to handle this.